rpcs3/Crypto/unpkg.cpp

```diff
@@ -45,7 +45,12 @@ bool CheckHeader(const fs::file& pkg_f, PKGHeader* m_header)
 	}
 	}
 
-	if (m_header->header_size != PKG_HEADER_SIZE)
+	if (m_header->header_size == PKG_HEADER_PS1_SIZE)
```

Comment:
http://www.emunewz.net/forum/showthread.php?tid=163227&pid=224215#pid224215 - the guy tried to install a PS1 .pkg file.